services/tables/src/main/java/com/linkedin/openhouse/tables/api/spec/v0/request/components/Replication.java

```diff
@@ -0,0 +1,27 @@
+package com.linkedin.openhouse.tables.api.spec.v0.request.components;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Builder(toBuilder = true)
+@EqualsAndHashCode
+@Getter
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
```

Comment:
Is this constructor needed? I think all args constructor would be used whenever we want to set replication policy.

Reply:
we need the `NoArgsConstructor` constructor as a default constructor otherwise `jackson` has issues instantiating the class for JSON deserialization in the tests